docs/storage-drivers/gcs.md

```diff
@@ -44,6 +44,17 @@ An implementation of the `storagedriver.StorageDriver` interface which uses Goog
       A private service account key file in JSON format. Instead of a key file <a href="https://developers.google.com/identity/protocols/application-default-credentials">Google Application Default Credentials</a> can be used.
     </td>
   </tr>
+  <tr>
+    <td>
+      <code>credentials</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+      The content of the JSON format key file. The keyfile argument has higher priority than the credentials argument if both are specified.
```

Comment:
Really? why when it can just work? Also I'm pretty sure no one will supply both, since they'd probably notice they're the same thing.